artichoke-backend/src/extn/core/kernel/require.rs

```diff
@@ -162,4 +166,65 @@ pub mod method {
         }
         Err(Error::CannotLoad(args.file.to_owned()))
     }
+
+    fn load_impl(interp: &Artichoke, args: Args, base: &str) -> Result<Require, Error> {
+        let interp = interp.clone();
+        let mut path = PathBuf::from(args.file);
+        let files = if path.is_relative() {
+            path = PathBuf::from(base);
+            let mut files = Vec::with_capacity(2);
+            if !args.file.ends_with(".rb") {
+                files.push(path.join(format!("{}.rb", args.file)))
```

Comment:
`Kernel#load` does not append `.rb` to the given filename to search.

```
[2.6.3] > load 'goo'
Traceback (most recent call last):
        5: from /usr/local/var/rbenv/versions/2.6.3/bin/irb:23:in `<main>'
        4: from /usr/local/var/rbenv/versions/2.6.3/bin/irb:23:in `load'
        3: from /usr/local/var/rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        2: from (irb):1
        1: from (irb):1:in `load'
LoadError (cannot load such file -- goo)
[2.6.3] > load 'goo.rb'
=> true
```

Reply:
Is this needed? I know load is similar to require, but I believe by default load doesn't work unless you provide the .rb extension.

Reply:
Yes that is what I am saying. This code adds an additional path to the list of paths to attempt to load by appending a `.rb` extension. `Kernel#load` should not add an `.rb` extension to the files it searches for. It should only attempt to load the given filename.

Reply:
Ok, then, if that is so then this is gone too.